html/docs/first_time.html

```diff
@@ -315,8 +315,8 @@ <h4><span class="label label-success">5</span> Commit the Change</h4>
 
             <pre>$ cd taskwarrior.git
 $ git status
-On branch 2.6.0
-Your branch is up-to-date with 'origin/2.6.0'.
```

Comment:
These are technically incorrect as we are now continously running on the `develop` branch, but it should not block this MR.